pages/confirm/[token].tsx

```diff
@@ -16,9 +17,9 @@ const initialValues = {
 
 const ConfirmSuccess: React.FC = () => (
   <Card type="success" title="Password has been set!">
-    <a className="btn btn-primary btn-lg mb-3" role="button" href="/curriculum">
```

Comment:
It seems like screen readers should view this as a link rather than a button, so I removed `role="button"`.